app/serializers/serialize_doc.rb

```diff
@@ -0,0 +1,22 @@
+class SerializeDoc
+  include Mandate
+
+  initialize_with :doc
+
+  def call
```

Comment:
I was unsure which data we want to display on the search page, so I've just gathered some data. Let me know if you'd like more (or less) data.

Reply:
Yep - this looks good I think.